src/components/inputs/text/NameTokenSelector/styles.css

```diff
@@ -0,0 +1,99 @@
+.custom-next-pagination {
+  display: 'flex';
+  justify-content: space-between;
+  width: 100%;
+}
+.custom-next-pagination > ul > li {
+  display: flex !important;
```

Comment:
Just a comment, not requesting changes: Be careful when using `!important` because it will overwrite any other style that matches the selector and may be messy later on. The way of making a CSS rule to be applied over others is to have a more specific CSS selector.

Reply:
Right - i think it might be necessary here because of how ant design controls their styles using the :where selector. I tried multiple different methods, but this should *only* modify those elements belonging to the custom-next-pagination class.

Reply:
fixed in https://github.com/ar-io/pdns-react/pull/229/commits/4de5520d092e8e98eaa9dcffe69150aac929361b

Reply:
Nothing to change here 👍